src/common/components/select-repository-list/index.js

```diff
@@ -34,7 +34,8 @@ export class SelectRepositoryListComponent extends Component {
     super();
 
     this.state = {
-      showMissingReposInfo: false
+      showMissingReposInfo: false,
+      addTriggered: false,
```

Comment:
Fixes issues reported in #855 - this flag in state allows us to only show spinner in Add button when it's clicked (repos are being added), not during initial loading.